proposal/ycchan5566/README.md

```diff
@@ -0,0 +1,56 @@
+# Proposal for Physics Engine
+
+## Basic information
+
+Implement a c++ base physics engine and use python to write a easy physical simulator.
+
+The GitHub repository:
+https://github.com/ycchan5566/Physics-Engine
+
+## Problem to solve
+
+Today, the number of physics engine provided by python is still small, so I decide to develop a physics engine in order to enrich the physical library of python.
+
+## Perspective users
+
+1. People who want to develop a game.
+2. People who want to simulate physical phenomenon.
+
+## System architecture
+
+1. Read the input (such as: mass, velocity) from users.
+2. Compute the result of collision and displacement.
```

Comment:
Are collision and displacement the only things you want to simulate?  In this way you are trying to build a kinetic engine.  Are there other things to be simulated?